chatkit-core/src/main/kotlin/com/pusher/chatkit/rooms/RoomSubscriptionEvent.kt

```diff
@@ -12,8 +12,7 @@ internal typealias RoomSubscriptionConsumer = (RoomSubscriptionEvent) -> Unit
  * Contrast with RoomEvent, the user facing events which are related to a room.
  */
 internal sealed class RoomSubscriptionEvent {
-    data class NewMessage(val message: Message) : RoomSubscriptionEvent()
-    data class NewMultipartMessage(val message: V3MessageBody) : RoomSubscriptionEvent()
+    data class NewMultipartMessage(val message: MessageApiType) : RoomSubscriptionEvent()
```

Comment:
this event can be now simply renamed to NewMessage

Reply:
👍 I also deleted the onErrorOccurred as it wasn't being used.